### DISCUSSION TITLE
Getting C-i to work as expected in tmux

### DISCUSSION THREAD

**Discussion by blancpain** [tmux/tmux#4172] [General]
Hi,

I am using tmux 3.5a running on wezterm. Previously I had the following workaround to get C-i to work as expected and to unlink it from "Tab" (ideas taken from here: https://github.com/tmux/tmux/issues/2705#issuecomment-1518520942-1518520942):

tmux.conf:
```
bind-key -T root F12 set key-table virt

bind-key -T virt 'C-i' if-shell "$is_vim" "send-keys Escape '[105;5u'" "select-pane -U" \; set key-table root
```

wezterm.lua:

```
config.keys = {
	{
		key = "i",
		mods = "CTRL",
		action = wezterm.action({ SendString = "\x1b[24~\x09" }),
	},
}
```

The above no longer works and as I understand a recent tmux update decoupled C-i from Tab. I expected this change means I actually don't need the above workaround but C-i still seems to be linked to Tab even if running without the configs above. 

Has anyone been able to resolve this?

**Comment by nicm** [comment]
Did you turn extended-keys on?

This should still work anyway because F12 has not changed, although I'm not sure what you ever needed the extra key table for.

**Comment by blancpain** [comment]
I hadn't turn on extended-keys - just tried and it doesn't work still.

I'm going with a dirty solution now where I map C-i to C-PageUp in wezterm and then map C-PageUp to C-i in nvim as a workaround.

**Reply by nicm** [reply]
OK well doesn't work isn't very helpful, if you think it is a genuine bug then please open an issue with logs.

**Reply by blancpain** [reply]
Fair enough, that's why I opened a discussion as don't have enough time ATM to properly debug. Will need to test further and open an issue if needed. In the meantime, the workaround above works fine for now.
